gen3-cli/cot/command/create/template.py

```diff
@@ -2,6 +2,7 @@
 import subprocess
 import click
 from cot import env
+from cot import utils
```

Comment:
multiple flake8 warning for this file, most of them are line width (which we should increase from standard 80 characters let's say to 100 - so it fits github UI), but one is about  forgotten import